code/procs/scanprocs.dm

```diff
@@ -258,7 +257,9 @@
 		return "<br><span style='color:purple'><b>[O.name]</b> - Moderate</span>"
 	else if (damage > 0)
 		return "<br><span style='color:purple'><b>[O.name]</b> - Minor</span>"
-
+	else
+		if (O.robotic)
+			return "<br><span style='color:purple'><b>[O.name]</b> - Robotic organ detected</span>"
```

Comment:
Not sure I like this that much, codewise. Right now, it only prints that if it's a robotic organ if it has no damage. I think it'd be useful to append robotic status to damaged organs as well. You could incorporate this by checking for robotics after damage stuff (change return to . = ""), append a robotic snippet to a previously generated text. However, if there was no previously generated text, you'' need to return the full organ information as well

Reply:
I was wondering if this was necessary because the organ name would already indicate it but it is inconsistent right now and appending it would be more consistent,